### ISSUE TITLE
[Suggestion] Crash Descriptions

### ISSUE THREAD

**Issue by D-a-r-n-o-l-d** [AntiCope/meteor-crash-addon#29]
### Describe the feature

add a small on-hover description describing when each crash is usable for dum-dums like me who don't know each method

**Comment by crazymoose7** [comment]
Well they do a have a small description of what they do when you hover over. But I assume you are talking about a description of the use cases for each crash. Maybe a small wiki on here could be useful which could contain that and more information on the module.

**Comment by D-a-r-n-o-l-d** [comment]
agreed.

I don't follow crashing methods for all server softwares and versions and I doubt Im the only one.

**Comment by crazymoose7** [comment]
> agreed.
> 
> I don't follow crashing methods for all server softwares and versions and I doubt Im the only one.

So just some information about where/what version each crash can be used on / what it does ?
I guess there really isnt thing anymore than that

**Comment by D-a-r-n-o-l-d** [comment]
Yeah just in what cases can I use each method so I know when a server is vulnerable
